### PR TITLE
feat: Add slack support.

### LINKED ISSUE #159: Feature: Add slack plugin to receive messages.
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've discussed this feature request in the [OpenIMSDK Slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) and got positive feedback

### Is this feature request related to a problem?

❎ No

### Problem Description

_No response_

### Solution Description

Add slack app and interact with Socket(debug mode)

Refer packages:
```bash
github.com/shomali11/slacker v1.4.1 // indirect
```

### Benefits

Support slack platform.

### Potential Drawbacks

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by IRONICBo**
#### 🔍 What type of PR is this?

/kind documentation

#### 👀 What this PR does / why we need it:

Use `slacker` to listen slack messages.

#### 🅰 Which issue(s) this PR fixes:

Fixes #159

### CODE REVIEW

File: server/internal/msg/slack_cmd/slack.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+	"github.com/shomali11/slacker"
+
+	"github.com/OpenIMSDK/OpenKF/server/internal/config"
+	"github.com/OpenIMSDK/OpenKF/server/internal/service"
+	"github.com/OpenIMSDK/OpenKF/server/pkg/log"
+)
+
+// InitSlack init slack client.
+func InitSlack() {
```
Comment (cubxxw) [inline server/internal/msg/slack_cmd/slack.go:29]: Only initialize Slack, and should not include the ability to listen or process commands.

**Review by cubxxw — APPROVED** [review]

File: server/internal/msg/slack_cmd/slack.go (hunk 1 above)
Comment (IRONICBo) [reply server/internal/msg/slack_cmd/slack.go:29]: Ok, I will update soon.

File: server/internal/service/slack.go [hunk 2]
```diff
@@ -46,3 +61,76 @@ func (svc *SlackService) GetSlackConfig() (*responseparams.SlackConfigResponse,
...
+		DisplayNameNormalized: profile.DisplayNameNormalized,
+		Skype:                 profile.Skype,
+		Phone:                 profile.Phone,
+		Image24:               profile.Image24,
+		Image32:               profile.Image32,
+		Image48:               profile.Image48,
+		Image72:               profile.Image72,
+		Image192:              profile.Image192,
+		Image512:              profile.Image512,
+		ImageOriginal:         profile.ImageOriginal,
```
Comment (charles-chenzz) [inline server/internal/service/slack.go:102]: the image number here is pixel?

Comment (IRONICBo) [reply server/internal/service/slack.go:102]: Yes, these fields are from github.com/slack-go/slack@v0.12.1/users.go:UserProfile, I save them to local OpenKF storage.

**Review by cubxxw — APPROVED** [review]

**Review by hanzhixiao — APPROVED** [review]

**Pull request merged**